app/assets/javascripts/admin/addon/models/report.js

```diff
@@ -503,7 +503,78 @@ const Report = EmberObject.extend({
   },
 });
 
+function computeGrouping(data, grouping) {
+  if (!grouping) {
```

Comment:
This seems to share some code with `_unitForDatapoints` above. Could it be DRYed up?

Reply:
It mostly share the constants, so yes maybe I could use constants